util/process.go

```diff
@@ -38,19 +39,40 @@ func ExecAndCaptureOutput(command string, args ...string) (string, string, error
 	subcmd.Stdout = wStdout
 	subcmd.Stderr = wStderr
 
-	err = subcmd.Run()
+	outputStdout := make([]byte, 0, 10240)
+	outputStderr := make([]byte, 0, 10240)
```

Comment:
given that we don't expect too much errors ( hopefully ), I would set it to a lower number. it will grow dynamically anyway.

Reply:
done